views/partials/header.ejs

```diff
@@ -3,9 +3,9 @@
 <meta http-equiv="X-UA-Compatible" content="IE=edge;chrome=1" />
 <script src="https://cdn.jsdelivr.net/npm/es6-promise@4/dist/es6-promise.auto.min.js"></script>
 <script src="https://unpkg.com/event-source-polyfill@0.0.12/src/eventsource.min.js"></script>
-<script src="https://app.launchdarkly.com/snippet/ldclient.min.js"></script>
+<script src="https://unpkg.com/launchdarkly-js-client-sdk@3"></script>
```

Comment:
This isn't ideal, but at least consistent with hello-js, and it gets things running. 

We  really need to redo all the JS examples with more realistic apps.